shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagedBridgeServiceImpl.java

```diff
@@ -226,6 +245,8 @@ public AuthorizationPolicy fetchOrCreateBridgeAuthorizationPolicy(ManagedBridge
         expected.getSpec().getSelector().setMatchLabels(Collections.singletonMap(Constants.BRIDGE_INGRESS_AUTHORIZATION_POLICY_SELECTOR_LABEL,
                 istioGatewayProvider.getIstioGatewayService().getMetadata().getLabels().get(Constants.BRIDGE_INGRESS_AUTHORIZATION_POLICY_SELECTOR_LABEL)));
 
+        ensureManagedBridgeLabels(managedBridge, expected);
```

Comment:
Nitpick: move this call to line 233 close to where the metadata is set for the `expected` resource? Same for all the other calls to `ensureManagedBridgeLabels` in the other methods

Reply:
Done 👍